client/src/components/pages/admin/DataVisualizationPage.js

```diff
@@ -0,0 +1,373 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import Chart1 from './LineChart';
+import ProgressBar from 'react-bootstrap/lib/Progressbar';
+import Chart2 from './DonutChart';
+
+import { getUsers, getAlerts} from 'actions';
+
+require('./tableStyles.scss');
+
+
+class DataVisualizationPage extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.defaultState = {
+            width: 0,
+            
+        }
+
+        this.state = this.defaultState;
+    }
+
+    componentWillMount() {
+        this.props.getUsers();
+        this.props.getAlerts();
+    }
+
+    filterAdvocates(users) {
+        let advocates = users.filter(function(user) { return user.role == "advocate"});
+        return advocates;
+    }
+
+    filterAdmin(users){
+        let admin = users.filter(function(user) { return user.role == "admin"});
+        return admin;
+    }
+
+    filterProviders(users){
+        let providers = users.filter(function(user) { return user.role == "providers"});
+        return providers;
+    }
+
+    alertAmountPrevious(alerts){
+        console.log(alerts[0])
+        var currentTime = new Date();
+
+    }
+
+    render() {
+        let {users, current_user, alerts} = this.props;
+        let advocates = this.filterAdvocates(users);
+        let providers = this.filterProviders(users);
+        let admin = this.filterAdmin(users);
+
+        let advocatesAmount = advocates.length;
+        let providersAmount = providers.length;
+        let alertAmount = alert.length;
+        let alertAmountPrevious = this.alertAmountPrevious(alerts);
+
+        var dashboard;
+        dashboard = (
```

Comment:
Declaring variables like this is redundant. Simply do `let dashboard = ...`.

Also we are not using the term "dashboard" in the project anymore.